crates/pathfinder/src/state.rs

```diff
@@ -418,8 +663,19 @@ mod tests {
         assert_eq!(result, expected);
     }
 
+    /*
+    use crate::{
+        core::{
+            EthereumBlockHash, EthereumBlockNumber, EthereumLogIndex, EthereumTransactionHash,
+            EthereumTransactionIndex, StarknetBlockHash, StarknetBlockNumber,
+        },
+        ethereum::test::create_test_transport,
+    };
+    use std::str::FromStr;
+    use web3::types::H256;
```

Comment:
This broke completly because I removed the unused `use`'s from the `super` as this was leaning on `use super::*;`

Reply:
Somewhat restored this.